src/moin/scripts/migration/moin19/_utils19.py

```diff
@@ -126,15 +126,11 @@ def unquoteWikiname(filename, charset=CHARSET):
     :rtype: unicode
     :returns: WikiName
     """
-    # From some places we get called with Unicode strings
-    if isinstance(filename, str):
-        filename = filename.encode(CHARSET)
```

Comment:
if this normalization is removed, shouldn't there be at least a type assertion here, so we can be sure with what we deal here?

Reply:
unquoteWikiname method was copied from wikiutil.py in 1.9 where is was called from several places. Here it is called from only 2 places within import19.py and both places pass string(unicode).

Reply:
ok